helm_resource/Tiltfile

```diff
@@ -105,10 +107,10 @@ helm uninstall %(namespace_arg)s %(release_name)s || true
     live_update=live_update)
 
   if resource_deps:
-    k8s_resource(name, resource_deps=resource_deps)
+    k8s_resource(name, resource_deps=resource_deps, port_forwards=port_forwards)
```

Comment:
can you change this to:

```
if len(port_forwards):
  k8s_resource(name, port_forwards=port_forwards)
```

otherwise, they might get added twice, which is not what you want

Reply:
Thank you for pointing that out